.github/workflows/build-test.yml

```diff
@@ -371,9 +371,9 @@ jobs:
       run: rustup show
 
     # Job-specific dependencies
-    - uses: dart-lang/setup-dart@v1
+    - uses: dart-lang/setup-dart@e51d8e571e22473a2ddebf0ef8a2123f0ab2c02c
       with:
-        sdk: 3.4.0-204.0.dev
+        sdk: stable
```

Comment:
thought: we pin a stable rust toolchain, we should probably also pin a stable dart toolchain.

Reply:
The downside of pinning is that you have to regularly manually update, as your customers are most likely not pinning. This way, the daily/weekly CI jobs catch any errors happening early. I would even recommend testing on `dev`, not or in addition to `stable`, for that exact reason.

Reply:
We don't use main CI as a canary, because then PRs will fail CI.

Reply:
Interesting. How would you want to test against the latest Dart version then? You could have to Github CI workflows, one only running on main, the other running against PRs.

Reply:
We're happy with updating the toolchain (and the build script) only when someone asks us to.

Reply:
If this package will be published sometime, that's testing in prod - you will notice failures only via bug reports/PRs. If it is not published, that would mean bitrot setting in. I would strongly suggest updating this to `dev` or at least `stable`. But I am also fine with pinning it at the current stable, so `3.7.0`, if you feel strongly about this.

Reply:
This package will not be published until native assets are stable, at which point there won't be breaking changes in every compiler version.

Reply:
Yes please pin 3.7.0